movies/presentation/src/main/java/com/example/movies/presentation/ui/movieDetail/MovieDetailFragment.kt

```diff
@@ -80,6 +84,8 @@ class MovieDetailFragment : BaseFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        //TODO implement viewModel call using SafeArg Argument passed through NavigationController
+        //TODO ie:  viewModel.getDetailMovie( args.movieId )
```

Comment:
Could you please implement this changes? send in the arguments to the MovieDetailFragment